src/utils/cleanIdentifierInputs.ts

```diff
@@ -1,10 +1,14 @@
 import { IdentifierInput } from "../types";
 
 export const cleanIdentifierInputs = (
-  input: IdentifierInput[]
+  input: IdentifierInput[],
+  optionalIdentifier: string[]
 ): IdentifierInput[] => {
   for (const identifierInput of input) {
     identifierInput.value = identifierInput.value.trim();
```

Comment:
We can make use of de-structuring here:

```suggestion
    const { value, label } = identifierInput;
    identifierInput.value = value.trim();
    if (optionalIdentifierLabels.includes(label)) {
```